zebra/main.c

```diff
@@ -437,8 +455,10 @@ int main(int argc, char **argv)
 	*  will be equal to the current getpid(). To know about such routes,
 	* we have to have route_read() called before.
 	*/
+	zrouter.startup = time(NULL);
```

Comment:
You need to update this to monotime after your other PR gets merged :)

Reply:
yep fixed.